lib/cadet_web/router.ex

```diff
@@ -140,6 +136,11 @@ defmodule CadetWeb.Router do
     post("/assets/:foldername/*filename", AdminAssetsController, :upload)
     delete("/assets/:foldername/*filename", AdminAssetsController, :delete)
 
+    post("/assessments", AdminAssessmentsController, :create)
+    post("/assessments/:assessmentid", AdminAssessmentsController, :update)
+    delete("/assessments/:assessmentid", AdminAssessmentsController, :delete)
+    post("/assessments/publish/:assessmentid", AdminAssessmentsController, :publish)
```

Comment:
This should be merged into `POST /assessments/:assessmentid` really&mdash;but if not, at least `/assessments/:assessmentid/published`.

(Routes should be nouns, as far as possible, because the verb is in the HTTP method i.e. `POST`)

Reply:
Right, I actually merged that endpoint into `POST /assessments/:assessmentid` but forgot to remove it from the router. Will update later.